src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Colors {

```

Comment:
class/enum names should be in singular
```suggestion
public enum Color {
```